### DISCUSSION TITLE
Joystick interface location

### DISCUSSION THREAD

**Discussion by stricklerlj61** [AntiMicroX/antimicrox#760] [Q&A]
Is it possible to direct the joystick search to another location? 

I have 4 joystick/button combinations of the same type (no serial number, Dragon Rise, ZeroDelay USB interfaces), the only difference is the USB port they are plugged into. I have used udev to create constant names in a different location (/dev/js0 thru /dev/js3) but can't find a way to make AntiMicroX to look there instead of /dev/input. 

QJoyPad allows a command line parameter to set where the search starts but I'd rather use AntiMicroX. 

The other potential would be for the uuid to include the connection path effectively doing the same thing, right now whichever one is recognized first gets the same uuid.

**Comment by pktiuk** [comment]
I don't think there is such an option.   
@mmmaisel was investigating solution which could potentially help with this https://github.com/AntiMicroX/antimicrox/pull/464
